src/pages/404.jsx

```diff
@@ -103,9 +159,22 @@ const NotFoundPage = () => {
       </TextureWrapper>
       <NotFound src={notFound} alt='404' />
 
+      {!isAprilFirst && (
+        <FishWrapper1>
```

Comment:
Éviter si possible d'utiliser des noms arbitrairea. Dans ce cas ci, `FishWrapperTop` ferait plus de sens. 
```suggestion
        <FishWrapperTop>
```